ts/features/it-wallet/screens/issuing/pid/cie/ItwPidPreviewScreen.tsx

```diff
@@ -58,89 +59,107 @@ const ItwPidPreviewScreen = () => {
+          <ForceScrollDownView>
             <VSpacer />
-            <ItwPidClaimsList
-              decodedPid={decodedPid}
-              claims={["givenName", "familyName", "taxIdCode"]}
-              expiryDate
-              securityLevel
-              onLinkPress={() =>
-                navigation.navigate(ITW_ROUTES.GENERIC.NOT_AVAILABLE)
-              }
-              issuerInfo
-            />
-            <VSpacer />
-            <H4 weight={"SemiBold"} color={"bluegreyDark"}>
-              {I18n.t(
-                "features.itWallet.verifiableCredentials.unrecognizedData.title"
-              )}
-            </H4>
-            <VSpacer />
-            <H4 weight={"Regular"} color={"bluegrey"}>
-              {I18n.t(
-                "features.itWallet.verifiableCredentials.unrecognizedData.body",
-                {
-                  issuer:
-                    decodedPid.pid.verification.evidence[0].record.source
-                      .organization_name
+            <View style={IOStyles.horizontalContentPadding}>
+              <H2>
+                {I18n.t("features.itWallet.issuing.pidPreviewScreen.title")}
+              </H2>
+              <VSpacer size={16} />
+              <Body>
+                {I18n.t(
+                  "features.itWallet.issuing.pidPreviewScreen.checkNotice"
+                )}
+              </Body>
+              <VSpacer size={24} />
+              <ItwCredentialCard
+                pidClaims={decodedPid.pid.claims}
+                display={pidDisplayData}
+              />
+              <VSpacer />
+              <ItwPidClaimsList
+                decodedPid={decodedPid}
+                claims={["givenName", "familyName", "taxIdCode"]}
+                expiryDate
+                securityLevel
+                onLinkPress={() => null}
+                issuerInfo
+              />
+              <VSpacer />
+              <Banner
+                color="neutral"
+                pictogramName="security"
+                title={I18n.t(
+                  "features.itWallet.issuing.pidPreviewScreen.banner.title"
+                )}
+                size="big"
+                content={I18n.t(
+                  "features.itWallet.issuing.pidPreviewScreen.banner.content"
+                )}
+                action={I18n.t(
+                  "features.itWallet.issuing.pidPreviewScreen.banner.actionTitle"
+                )}
+                onPress={() =>
+                  navigation.navigate(ITW_ROUTES.GENERIC.NOT_AVAILABLE)
                 }
-              )}
-            </H4>
-            <VSpacer />
-            <ButtonOutline
-              fullWidth
-              accessibilityLabel="ClamListButton"
-              label={I18n.t(
-                "features.itWallet.verifiableCredentials.unrecognizedData.cta"
-              )}
-              onPress={() =>
-                navigation.navigate(ITW_ROUTES.GENERIC.NOT_AVAILABLE)
-              }
+                onClose={constNull}
```

Comment:
I think this should be removed as it shows an "X" but the banner isn't dismissable in the FIGMA.